docs/DeveloperGuide.md

```diff
@@ -103,9 +103,10 @@ Here's a (partial) class diagram of the `Logic` component:
 
 How the `Logic` component works:
 1. When `Logic` is called upon to execute a command, it uses the `AddressBookParser` class to parse the user command.
-1. This results in a `Command` object (more precisely, an object of one of its subclasses e.g., `AddCommand`) which is executed by the `LogicManager`.
-1. The command can communicate with the `Model` when it is executed (e.g. to add a person).
-1. The result of the command execution is encapsulated as a `CommandResult` object which is returned back from `Logic`.
+2. If the view or add command is executed, in addition to the `AddressBookParser` class, next `Logic` uses the `ContactDetailsParser` class to parse the user command. 
```

Comment:
I think this suggests that user input is parsed by AddressBookParser and then ContactDetailsParser, when instead its either-or. Maybe instead we could say if a command is entered from the Home Page it goes to the AddressBookParser and if it is entered from the Contact Details Page it goes to the ContactDetailsParser. This section comes after the UI section so the reader should be familiar with the pages by this point.

Reply:
Gd point